app/services/analytics_service/src/middleware/monitoring_middleware.py

```
@@ -0,0 +1,330 @@
+import time
+import uuid
+from datetime import datetime, timezone
+from typing import Dict, Any, Optional
+from functools import wraps
+import structlog
+from flask import Flask, request, Response, g, has_request_context
```

Ruff (0.11.9)

`flask.Response` imported but unused

Remove unused import: `flask.Response`

(F401)